google_shopping_functions.js

```diff
@@ -1,25 +1,47 @@
 //define getItemsCount function
 //accepts full item data
 //returns the length of the items array
+
+//load products.json
+var data = require("./products.json")
+
 var getItemsCount = function(itemData){
 	return itemData.items.length;
 };
 
+// takes in data, gives back data
+
 //
 //DEFINE FUNCTIONS HERE
 //
+var getItems = function(objectData) {
+	return objectData.items
+};
+//Question 1^
 
+var items = getItems(data);
 
+var getItemsByBrand = function(items, brand) {
+//not sure of code
+};
 
+return; //not sure of the return, it's an array of some sort (or what to test below with console.log)
 
```

Comment:
You've got the basic structure of your functions here. Just remember that the return statement needs to go inside of the function, (before the '}', not after). Inside of your functions, you will need to loop through the 'items' that you are passing in in order to filter out the ones you want by brand, availability, etc.